websites/M/MangaLife/presence.ts

```diff
@@ -0,0 +1,133 @@
+const presence = new Presence({ clientId: "906057236927893576" }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const [privacy, cover, timestamps, buttons] = await Promise.all([
+    presence.getSetting<boolean>("privacy"),
+    presence.getSetting<boolean>("cover"),
+    presence.getSetting<boolean>("timestamps"),
+    presence.getSetting<boolean>("buttons")
+  ]);
+  let presenceData: PresenceData = {
+    largeImageKey: "logo",
+    smallImageKey: !privacy ? "home" : "",
+    details: "Browsing...",
```

Comment:
Browsing homepage instead of state and details here

Reply:
but i kinda reused the `details: Browsing...` for several other pages, so changing it from `Browsing...` to `Browsing Home Page...` can be a conflict